### ISSUE TITLE
Albert Einstein is CPU Hungry

### ISSUE THREAD

**Issue by vanishakesswani** [pulkomandy/critical-mass#7]
![critimass1](https://cloud.githubusercontent.com/assets/24365311/21418317/4f3903e6-c846-11e6-8c7f-cc3ea4601f4a.png)
![critimass2](https://cloud.githubusercontent.com/assets/24365311/21418318/4f3b57a4-c846-11e6-9493-b3c6207ab807.png)

These two screenshots show the my CPU usage against A.I and Albert Einstein .  As soon as I change my Blue Player to Albert the CPU hikes up .
Probably we need a super computer to process Albert's Brain quickly  :P

**Comment by pulkomandy** [comment]
Yes, it needs a lot of CPU. If you can win the game at all other AI levels, maybe you are smart enough to optimize the code for this one! (I can't!)
